### DISCUSSION TITLE
Request for the Mesmerizer board

### DISCUSSION THREAD

**Discussion by stanleyseow** [PlummersSoftwareLLC/NightDriverStrip#280] [General]
Hi Dave,

Just watch your video on the Mesmerizer ESP32, I would like to request for a copy of the Mesmerize board for the following reasons :-

- I love audio spectrum analyzer (see my TikTok links below )  and program them in as many platform as possile ( ESP32, Arduino + Processing, Pi Pico , Teensy, etc )
- have been messing around with WS2812 in many projects and recently those HUB75 boards ( mostly P2 64x64 boards )
- I contribute to open source projects
- I do mostly Vue.js for web frontend 
- I do a lot of large art installation projects too 

https://www.tiktok.com/@stanleyseow
https://github.com/stanleyseow

Hope to hear from you

Stanley

**Comment by rbergen** [comment]
I'm going to close this as a duplicate of #278. Please refer to the answer of that discussion (or otherwise, the top of the README) for instructions.
